### DISCUSSION TITLE
Filtering DAGs by tags

### DISCUSSION THREAD

**Discussion by WulkanHefajstos** [apache/airflow#29558] [General]
Hi!

I'm working in fairly large Airflow environment, with dozens of active contributors, hundreds of independent ETL pipelines and over 1k DAGs. To improve DAGs' discoverability in __Airflow UI__ we use tags extensively, for instance to add information about project name, data provider, etc.

At the moment, Airflow allows us to filter DAGs by one or more tags in a disjunctive fashion. So for instance, when i provide two tags: `tag1` and `tag2`, it will search for all DAGs that include either `tag1` or `tag2`. If i'm not mistaken, there's no way to perform conjunctive search, let's say to find all DAGs with the same 'vendor' tag and tag indicating that it's a step responsible for data ingestion (in the similar way that you can apply filters in GitHub using labels - find all issues marked as bug that affect let's say Airflow core code).

So in this context - I'm wondering if anyone else is facing similar 'problems' ? 🙂 

Do you know if  there's any way that we could apply conjunctive filter or if that even makes any sense ?

Cheers!

**Comment by potiuk** [comment]
Not possible today. but if you come up with nice PR providing that feature (and solves the UX /UI issue on how to do it) I am sure it will get seriously considered :)
